LICENSE

```diff
@@ -2,11 +2,11 @@ MIT License
 
 Copyright (c) 2016 Software Engineering Education - FOSS Resources
 
-Permission is hereby granted, free of charge, to any person obtaining a copy
+Permission is hereby granted, free of charge, to any stock obtaining a copy
```

Comment:
Why is all `person` changed to `stock`?

Reply:
We would be storing stocks rather than person in our warenager so I thought the renaming of the class would be appropriate here.